pacman_mirrors/pacman_mirrors.py

```diff
@@ -127,211 +147,361 @@ def parse_cmd(self):
         parser.add_argument("-c", "--country",
                             type=str,
                             help=_("comma separated list of countries "
-                                   "where mirrors will be used"))
+                                    "where mirrors will be used"))
         parser.add_argument("--geoip",
                             action="store_true",
                             help=_("detect country using geolocation, "
-                                   "ignored if '--country' is used"))
+                                    "ignored if \"--country\" is used"))
```

Comment:
try comparing 
`ignored if '--coutnry' is used`
with
`ignored if "--country" is used`

Grammatically double qoutes is used when you qoute a message relayed by a person

He said "Bring me that car" in opposition of qouting a wording like the above text